pyproject.toml

```
@@ -26,8 +26,7 @@ Source = "https://github.com/marshmallow-code/marshmallow-sqlalchemy"
 
 [project.optional-dependencies]
 docs = ["sphinx==7.4.5", "alabaster==1.0.0", "sphinx-issues==4.1.0"]
-# TODO: Remove pytest pin when https://github.com/TvoroG/pytest-lazy-fixture/issues/65 is resolved
-tests = ["pytest<8", "pytest-lazy-fixture>=0.6.2"]
+tests = ["pytest", "pytest-lazy-fixtures"]
```

We should still keep pytest pinned to the latest major version:

```suggestion
tests = ["pytest<9", "pytest-lazy-fixtures"]
```

---

Not sure why one should pin pytest to the latest major version, but I don't have any objections to this, so I'll commit the suggestion.
I didn't write the original version of the patch - that was originally written by my Debian colleague Alexandre Detiste <tchet@debian.org>, so I don't really regard myself as an author for this. I just tweaked it for pytest-lazy-fixtures 1.1.1.

---

> Not sure why one should pin pytest to the latest major version,

we pin the major version to prevent compat breakages when pytest releases a major update--admittedly rare, but it's helpful to catch these via the automated dependency update PRs.

> I didn't write the original version of the patch 

OK, i'll leave add your colleague to AUTHORS then